_includes/header.html

```diff
@@ -21,12 +21,16 @@
                         <ul class="address-list">
                             <li>
                                 <label>
-                                    <a href="mailto:info@deepnetwork.com"><i class="fas fa-envelope"></i> info@deepnetwork.com</a>
+                                    <a href="mailto:info@deepnetwork.com">
+                                    <svg class="headericon"><use xlink:href="../images/sprite.svg#icon-envelope"></use></svg>    
```

Comment:
I think this `<use xlink:href="` is new; I just needed to google it to understand. Do old browsers understand it as well?

Reply:
SVG's use tag has a usage support of %92 
https://caniuse.com/#feat=mdn-svg_elements_use
As you'll see below there is a polyfill